src/node/tendermint_data.mli

```diff
@@ -0,0 +1,115 @@
+(** Tendermint input_log and output_log.
+    Holds all the querying function on the input_log required by Tendermint subprocesses. *)
+
+open Crypto
+open Tendermint_internals
+
+type elmt = value * round
+
+module MySet : Set.S with type elt := elmt
+
+(* FIXME: ConsensusStep2 Tendermint we could do better *)
+type node_identifier = Key_hash.t
+
+type index_step = consensus_step
```

Comment:
I really prefer to avoid type aliases unless it's for the purpose of abstraction. Why add the extra mental pointer-lookup?